pyradise/process/registration.py

```diff
@@ -432,6 +443,7 @@ def __init__(
         smoothing_sigmas: Tuple[float, ...] = (2, 1, 0),
         sampling_percentage: float = 0.2,
         resampling_interpolator: int = sitk.sitkBSpline,
+        deterministic: bool = False,
```

Comment:
In my opinion, this should be true by default. What do you think, @melandur?

Reply:
true